diff --git a/node_modules/@uniswap/v3-periphery/contracts/libraries/PoolAddress.sol b/node_modules/@uniswap/v3-periphery/contracts/libraries/PoolAddress.sol
index 60de385..b6acc73 100644
--- a/node_modules/@uniswap/v3-periphery/contracts/libraries/PoolAddress.sol
+++ b/node_modules/@uniswap/v3-periphery/contracts/libraries/PoolAddress.sol
@@ -32,17 +32,11 @@ library PoolAddress {
     /// @return pool The contract address of the V3 pool
     function computeAddress(address factory, PoolKey memory key) internal pure returns (address pool) {
         require(key.token0 < key.token1);
-        pool = address(
-            uint256(
-                keccak256(
-                    abi.encodePacked(
-                        hex'ff',
-                        factory,
-                        keccak256(abi.encode(key.token0, key.token1, key.fee)),
-                        POOL_INIT_CODE_HASH
-                    )
-                )
-            )
-        );
+         pool = address(uint160(uint256(keccak256(abi.encodePacked(
+            hex'ff',
+            factory,
+            keccak256(abi.encode(key.token0, key.token1, key.fee)),
+            POOL_INIT_CODE_HASH
+        )))));
     }
 }
